server: re-enable TestAdminAPITableStats

tbg opened the pull request:
This was allegedly fixed in https://github.com/cockroachdb/cockroach/pull/9182
though it needed some brushing up when I re-enabled it.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/20625)
<!-- Reviewable:end -->

bdarnell commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Review status: 0 of 1 files reviewed at latest revision, all discussions resolved, some commit checks pending.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/20625#-:-L0669Z4TMCAorQEtrq5:bnfp4nl)*
<!-- Sent from Reviewable.io -->

tschottdorf commented:
Odd, I've been running on my azworker for well above an hour with the temporary second commit, but am not seeing any failures.

tschottdorf commented:
Filed https://github.com/cockroachdb/cockroach/issues/20764 to track this. Doesn't seem particular to this test, so I'm going to go ahead and merge this.

tbg committed:
server: re-enable TestAdminAPITableStats

This was allegedly fixed in https://github.com/cockroachdb/cockroach/pull/9182
though it needed some brushing up when I re-enabled it.

Release note: None
